addrmgr/network.go

```diff
@@ -259,8 +259,8 @@ func GroupKey(na *wire.NetAddress) string {
 		// teredo tunnels have the last 4 bytes as the v4 address XOR
 		// 0xff.
 		ip := net.IP(make([]byte, 4))
-		for i, byte := range na.IP[12:16] {
-			ip[i] = byte ^ 0xff
+		for i, byte_ := range na.IP[12:16] {
```

Comment:
Really small nitpick but generally Go identifiers to use underscores. What do you think about `ipByte` or `addressByte`?

Reply:
Yeah it will need to be changed to pass the linter